index.html

```diff
@@ -8,8 +8,11 @@
   <body>
     <div id="top">
       <a href="#Select+user" id="select_user" onclick='return page.selectUser()'>Select user</a>
-      <input type="file" onchange='return page.uploadSong(this)' />
-      <audio id="player" controls></audio>
+      <input type="file" onchange='return page.updateMetadata(this)' id="file"/>
+      artist<input type="text" id="artist" />
+      title<input type="text" id="title" />
+      <input type="button" value="upload" onclick="page.uploadSong(document.getElementById('file'))" />
+      <audio id="player" onended="playNext()" controls></audio>
```

Comment:
There is currently no `playNext` function, so this line modification is useless/non-working. I think you should remove the `onended` attribute.